parser.go

```diff
@@ -285,33 +299,60 @@ func (p *parser) readPythonMultilines(line string, bufferSize int) (string, erro
 	parserBufferPeekResult, _ := p.buf.Peek(bufferSize)
 	peekBuffer := bytes.NewBuffer(parserBufferPeekResult)
 
+	indentSize := 0
 	for {
 		peekData, peekErr := peekBuffer.ReadBytes('\n')
 		if peekErr != nil {
 			if peekErr == io.EOF {
+				p.debug("readPythonMultilines: failed to peek")
+				p.debug("readPythonMultilines: peekData is: '"+string(peekData)+"'")
+				p.debug("readPythonMultilines: value is: '"+line+"'")
 				return line, nil
 			}
+			p.debug("readPythonMultilines: failed to peek, returning error")
 			return "", peekErr
 		}
 
+		p.debug("readPythonMultilines: parsing '"+string(peekData)+"'")
+
 		peekMatches := pythonMultiline.FindStringSubmatch(string(peekData))
+		p.debug("readPythonMultilines: matched ", len(peekMatches), " parts:")
+		for n, v := range peekMatches {
+			p.debug("   ", n, ": '", v, "'")
+		}
```

Comment:
One single debug line should be self-contained, it can be very long and I don't think that will be a problem.

Reply:
The debug line in the for loop can't be expressed any simpler since at this point in the function it is unknown how many elements are in peekMatches.

Reply:
> since at this point in the function it is unknown how many elements are in peekMatches.

What do you mean by "unknown how many elements in peekMatches"? `len(peekMatches)`?

Reply:
I was referring to your comment that you want this to be in a single line. I responded saying that it can't be since I have to iterate over the number of matches, hence the for loop instead of a single line. If the number of matches was fixed, I could just write a single statement printing all at once.